packer/install-dependencies.sh

```diff
@@ -360,9 +360,8 @@ echo "  - busybox" >> ${RELEASE_NOTES_FILEPATH}
 
 # TODO: fetch supported k8s versions from an aks-engine command instead of hardcoding them here
 K8S_VERSIONS="
-1.14.2
+1.14.3
 1.14.1
-1.13.6
```

Comment:
Or I can remove this in the upcoming 1.13.7 PR if we prefer.

Reply:
I'm fine with either, the 1.13.7 PR makes more logical sense but it's not a big deal.